Formula/m/mpv.rb

```diff
@@ -19,6 +19,7 @@ class Mpv < Formula
   end
 
   depends_on "docutils" => :build
+  depends_on "libplacebo" => :build
```

Comment:
Failure means the library was dynamically linked, so `libplacebo` will need to be runtime dependency rather than build-only.

Reply:
Thank you. Fixed.